src/edu/csus/ecs/pc2/ui/FinalizePane.java

```diff
@@ -92,11 +97,11 @@ public FinalizePane() {
 
     /**
      * This method initializes this
-     * 
+     *
      */
     private void initialize() {
         this.setLayout(new BorderLayout());
-        this.setSize(new Dimension(457, 239));
+        this.setSize(new Dimension(560, 239));
```

Comment:
Could this be 240 instead of 239? Would be nice.

Reply:
I think this is a result of the window builder (GUI resource editor).  When resizing things in window builder, we often do not pay attention to the sizes of things.